graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/kotlin/KotlinDataTypeGenerator.kt

```diff
@@ -28,7 +28,7 @@ import graphql.language.*
 
 class KotlinDataTypeGenerator(private val config: CodeGenConfig, private val document: Document) : AbstractKotlinDataTypeGenerator(config.packageNameTypes, config) {
     fun generate(definition: ObjectTypeDefinition, extensions: List<ObjectTypeExtensionDefinition>): CodeGenResult {
-        if (definition.shouldSkip()) {
+        if (definition.shouldSkip(config)) {
```

Comment:
nit/repeat: the duplication of this block across the various generators resembles the duplication for the type definitions, in that they seem to lack a shared abstraction encapsulating their common operations